toolkit/cli/smart-contracts-commands/src/main.rs

```diff
@@ -0,0 +1,16 @@
+use clap::Parser;
+use partner_chains_smart_contracts_commands::SmartContractsCmd;
+
+#[derive(Clone, Debug, clap::Parser)]
+pub enum SmartContractsCmdStandalone {
+	#[clap(flatten)]
+	Inner(SmartContractsCmd),
+}
+
+type CmdResult<T> = Result<T, Box<dyn std::error::Error + Send + Sync>>;
+
+#[tokio::main]
+async fn main() -> CmdResult<()> {
```

Comment:
Do we need it? The plan was to have zero binaries. Users were supposed use their own node binary

Reply:
That's QoL for us, because the node crate takes a long time to rebuild during development.

Reply:
Okay, I won't block PR on this. I'll try to figure out some alternative, when I work on other commands